packages/inventory/src/services/reservation-item.ts

```diff
@@ -131,38 +131,44 @@ export default class ReservationItemService {
    */
   @InjectEntityManager()
   async create(
-    data: CreateReservationItemInput,
+    data: CreateReservationItemInput | CreateReservationItemInput[],
```

Comment:
comment: you did it here, so my previous question becomes relevant

Reply:
this is just legacy, it will always be called with an array. also not part of the public api